doc/dev/source/installation.rst

```diff
@@ -19,6 +19,8 @@ You will need some system libraries:
 * *libffi-devel*
 * *openldap-devel* (if you wish to use LDAP authentication)
 
+You will also need the GNU Compiler Collection (commonly known by the package name gcc).
```

Comment:
What about something like this?

> Since some of the python dependencies contain compiled libraries you need a compiler such as gcc.

I don't think anyone really uses the term "GNU Compiler Collection" instead of just "gcc".